config/runtime.exs

```diff
@@ -8,15 +8,31 @@ switches = [
   mock_execution: :boolean,
   mode: :string,
   datadir: :string,
+  metrics: :string,
```

Comment:
its counterintuitive for me that it is called metrics but receives a port. Maybe two config options would be clearer?

Reply:
Sure!